tests/install_hooks_test.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+import subprocess
+
+from git import Repo
+from rdm.cli import install_hooks
+
+
+@pytest.fixture
+def tmp_repo(tmpdir):
+    directory = str(tmpdir)
+    repo = Repo.init(directory)
+    os.chdir(directory)
+
+    yield repo
+
+    # teardown
```

Comment:
Do you think this is a worthwhile comment?

Reply:
I was using it just to keep track of the code flow when I was still testing how a fixture worked, but the teardown itself is pretty straightforward consisting of a single line so I'll delete it.